test/e2e/vue.test.js

```diff
@@ -1,141 +1,206 @@
-          },
-          scriptURLs: vueURL,
-        };
+      test(`ignores content when Vue is not present`, async () => {
+        const docsifyInitConfig = getSharedConfig();
 
         await docsifyInit(docsifyInitConfig);
+        await page.evaluate(() => {
+          return 'Vue' in window === false;
+        });
+        await expect(page).toEqualText('#vuefor', '{{ i }}');
+        await expect(page).toEqualText('#vuecomponent', '---');
+        await expect(page).toEqualText('#vueglobaloptions p', '---');
+        await expect(page).toEqualText('#vuemounts p', '---');
+        await expect(page).toEqualText('#vuescript p', '---');
+      });
+
+      test(`ignores content when vueComponents, vueMounts, and vueGlobalOptions are undefined`, async () => {
+        const docsifyInitConfig = getSharedConfig();
+
+        docsifyInitConfig.config.vueComponents = undefined;
+        docsifyInitConfig.config.vueGlobalOptions = undefined;
+        docsifyInitConfig.config.vueMounts = undefined;
+        docsifyInitConfig.scriptURLs = vueURL;
 
-        const textContent = await page.textContent('#test span');
-        expect(textContent).toBe('');
+        await docsifyInit(docsifyInitConfig);
+        await expect(page).toEqualText('#vuefor', '{{ i }}');
+        await expect(page).toEqualText('#vuecomponent', '---');
+        await expect(page).toEqualText('#vueglobaloptions p', '---');
+        await expect(page).toEqualText('#vuemounts p', '---');
+        await expect(page).toEqualText('#vuescript p', 'vuescript');
       });
-    }
-  });
+
+      test(`ignores content when vueGlobalOptions is undefined`, async () => {
+        const docsifyInitConfig = getSharedConfig();
+
+        docsifyInitConfig.config.vueGlobalOptions = undefined;
+        docsifyInitConfig.scriptURLs = vueURL;
+
+        await docsifyInit(docsifyInitConfig);
+        await expect(page).toEqualText('#vuefor', '12345');
+        await expect(page).toEqualText('#vuecomponent', '0');
+        expect(await page.innerText('#vueglobaloptions p')).toBe('');
+        await expect(page).toEqualText('#vuemounts p', 'vuemounts');
+        await expect(page).toEqualText('#vuescript p', 'vuescript');
+      });
+
+      test(`ignores content when vueMounts is undefined`, async () => {
+        const docsifyInitConfig = getSharedConfig();
+
+        docsifyInitConfig.config.vueMounts['#vuemounts'] = undefined;
+        docsifyInitConfig.scriptURLs = vueURL;
+
+        await docsifyInit(docsifyInitConfig);
+        await expect(page).toEqualText('#vuefor', '12345');
+        await expect(page).toEqualText('#vuecomponent', '0');
+        await expect(page).toEqualText(
+          '#vueglobaloptions p',
+          'vueglobaloptions'
+        );
+        await expect(page).toEqualText('#vuemounts p', 'vueglobaloptions');
+        await expect(page).toEqualText('#vuescript p', 'vuescript');
+      });
+
+      test(`ignores <script> when executeScript is false`, async () => {
```

Comment:
This test doesn't seem to test that scripts were ignored, but that Vue still ran. Should the description be `renders Vue even if executeScript is false` or something similar?

Reply:
It's working as expected, but I can understand why the title could be confusing. 

Here is the relevant markup being tested:

```html
<div id="vuescript">
  <p v-text="msg">---</p>
</div>
```

* When the markdown `<script>` is executed
  * `#vuescript` is mounted directly by the `<script>`
  * The text content of `<p v-text="msg">` is changed to `"vuescript"`
* When the markdown `<script>` _not_ is executed
  * `#vuescript` is auto-mounted by docsify using `vueGlobalOptions`
  * The text content of `<p v-text="msg">` is changed to `"vueglobaloptions"`

That's why this test passes:

```js
await expect(page).toEqualText('#vuescript p', 'vueglobaloptions');
```